client/assets/stylesheets/shared/_reset.scss

```diff
@@ -123,6 +123,16 @@ a:active {
 a img {
 	border: 0;
 }
+button {
+	-webkit-appearance: none;
```

Comment:
Do we need this? Doesn't the `autoprefixer` PostCSS plugin generate this for us?

Reply:
You're right, it's not needed. Without it autoprefixer already produces:
```css
  -webkit-appearance: none;
  -moz-appearance: none;
       appearance: none;
```

Removed in https://github.com/Automattic/wp-calypso/pull/37026/commits/c2e9fd4da64ccaa7047dfa0b1c3949325e08ca52 and https://github.com/Automattic/wp-calypso/pull/37026/commits/1ea0288894923f7b44717b74f7c9249899b6c607